src/aws-manager.js

```diff
@@ -194,6 +194,8 @@ class AwsManager {
     assert(maxInstanceLife);
     assert(influx);
     assert(monitor);
+    assert(describeInstanceDelay);
+    assert(describeSpotInstanceDelay);
```

Comment:
I might be missing something, but I'm not really sure where these variables are being declared